rtt_roscomm/include/rtt_roscomm/rtt_rostopic_ros_msg_transporter.hpp

```diff
@@ -241,6 +242,7 @@ namespace rtt_roscomm {
     ros::NodeHandle ros_node;
     ros::NodeHandle ros_node_private;
     ros::Subscriber ros_sub;
+    PassthroughCallbackQueue prassthrough_callback_queue;
```

Comment:
Wrong order: The `PassthroughCallbackQueue` must exist before `ros::Subscriber`, such that it is destructed after the subscriber that uses it. Or the subscriber needs to be shutdown explicitly in the `~RosSubChannelElement()` destructor before the callback queue gets destroyed. But like this the subscriber refers to a dangling callback queue instance during destruction.

Probably calling `ros_sub.shutdown()` in the destructor would be safest.